scripts/test/test-utils.sh

```diff
@@ -154,5 +156,18 @@ get_pytest_files() {
   wget -N ${MENDER_ACCEPTANCE_URL}/helpers.py -P $WORKSPACE/mender-image-tests
   wget -N ${MENDER_ACCEPTANCE_URL}/conftest.py -P $WORKSPACE/mender-image-tests
   wget -N ${MENDER_ACCEPTANCE_URL}/fixtures.py -P $WORKSPACE/mender-image-tests
+  mkdir -p $WORKSPACE/mender-image-tests/files
+  wget -N ${MENDER_ACCEPTANCE_URL}/files/test-private-EC.pem -P $WORKSPACE/mender-image-tests/files
+  wget -N ${MENDER_ACCEPTANCE_URL}/files/test-private-RSA.pem -P $WORKSPACE/mender-image-tests/files
+  wget -N ${MENDER_ACCEPTANCE_URL}/files/test-public-EC.pem -P $WORKSPACE/mender-image-tests/files
+  wget -N ${MENDER_ACCEPTANCE_URL}/files/test-public-RSA.pem -P $WORKSPACE/mender-image-tests/files
```

Comment:
suggested refactoring:
```bash
for a in EC RSA; do
  for t in public private; do
    wget -N ${MENDER_ACCEPTANCE_URL}/files/test-${t}-${a}.pem -P $WORKSPACE/mender-image-tests/files;
  done;
done;
```

Reply:
Nice!

I originally wanted to refactor and move these keys into the `mender-image-tests` repository, but run into trouble with meta-mender and decided to just copy. I kind of "liked" that it was explicit as to point out to someone in the future that this is very ugly to do and we should refactor instead...

Anyway, I'll do your suggested rework.

Reply:
Thinking about it again, sorry but your suggestion is technically longer (5 lines) and adds complexity. So I'll leave it plain and simple for the reason stated above (that we know this full `get_pytest_files` idea is a horrible one).